src/rest-server/src/config/vc.js

```diff
@@ -37,8 +40,47 @@ const vcStatusPutInputSchema = Joi.object().keys({
     .required(),
 }).required();
 
+let resourceUnits;
+if (hivedConfig.enabledHived) {
+  resourceUnits = yaml.safeLoad(fs.readFileSync(hivedConfig.hivedSpecPath)).physicalCluster.cellTypes.leaves;
+} else {
+  resourceUnits = {
+    DEFAULT: {
+      gpu: 1,
+      cpu: 4,
+      memory: '8192Mi',
+    },
+  };
+}
+
+const convertMemory = (memoryStr) => {
+  let memoryMb = parseInt(memoryStr);
+  switch (memoryStr.replace(/[0-9]/g, '')) {
+    case 'Ti':
+      memoryMb *= 1000000;
+      break;
+    case 'Gi':
+      memoryMb *= 1000;
+      break;
+    case 'Mi':
+      break;
+    case 'Ki':
+      memoryMb /= 1000;
+      break;
+    default:
+      memoryMb /= 1000000;
+  }
+  return memoryMb;
+};
+
+for (let gpuType of Object.keys(resourceUnits)) {
+  resourceUnits[gpuType].memoryMB = convertMemory(resourceUnits[gpuType].memory);
```

Comment:
why cannot use `memoryMB` directly?

Reply:
It's defined as k8s memory style of `Gi, Mi`, not MB